### DISCUSSION TITLE
Default form actions alignment

### DISCUSSION THREAD

**Discussion by ariesdg** [filamentphp/filament#9500] [Help] [answered]
Hi, I'd just like to ask, in Filament V2 there's a filament config file where you can align the default form action, would it still be available in V3?

```
 'forms' => [
            'actions' => [
                'alignment' => 'right',
            ],
            'have_inline_labels' => false,
        ],
```

**Comment by VincvanAs** [comment]
As far as i can tell you can't set a default alignment. You can use e.g.

`public static string | Alignment $formActionsAlignment = Alignment::Right;
`
On create/edit pages etc.

**Reply by ariesdg** [reply]
This works as well, thank you

**Comment by leandrocfe** [comment, marked as answer]
Add in the `app/Providers/AppServiceProvider.php`

```php
use Filament\Pages\Page;
use Filament\Support\Enums\Alignment;

public function boot(): void
{
    Page::formActionsAlignment(Alignment::Right);
}
```

**Reply by ariesdg** [reply]
Thanks, it worked

**Reply by PetroZaburko** [reply]
How I can do the same with modal windows? I use some simple (modal) resources. Please help.

**Reply by reense** [reply]
@PetroZaburko have you ever found out how to do that?

**Reply by leandrocfe** [reply]
```php
MountableAction::configureUsing(function (MountableAction $action) {
    $action->modalFooterActionsAlignment(Alignment::Right);
});
```

?

**Reply by PetroZaburko** [reply]
That's it! @leandrocfe You are awesome!

**Reply by leandrocfe** [reply]
Sorry for the delay ;)

**Reply by aphilipdp** [reply]
> ```
> MountableAction::configureUsing(function (MountableAction $action) {
>     $action->modalFooterActionsAlignment(Alignment::Right);
> });
> ```
> 
> ?

Where to put this code? sorry im new

**Reply by PetroZaburko** [reply]
In boot method of any registered ServiceProvider
